src/hooks/useCds/translate.js

```diff
@@ -231,12 +232,83 @@ const snomedPregnancyCare = {
   'display': 'Antenatal care (regime/therapy)'
 };
 
+const immunosuppressedObservation = {
+  label: 'Immunosuppressed',
+  code: {
+    coding: [
+      {
+       system: SCT_URL,
+        code: '370388006',
+        display: 'Patient immunocompromised (finding)'
+      }
+    ]
+  }
+};
+
+const pregnantObservation = {
```

Comment:
The way we have pregnancy Observation defined in the CQL, the code you used here is the Observation.valueCodeableConcept. Need "Pregnancy status" as the Observation.code.

I'm open to re-reviewing how it is represented, but we'll need to stay consistent between the observations added here and the CQL. See https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/main/cql/DashboardLibrary.cql#L513-L520.